### PR TITLE
Corrected script for some item #16

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_combos.yml [hunk 1]
```diff
@@ -32236,3 +32252,1072 @@ Body:
...
+    Script: |
+      .@r = getequiprefinerycnt(EQI_SHOES);
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer)))
+         bonus bPerfectHitAddRate,3*.@r;
+  - Combos:
+      - Combo:
+          - aegis_29725    # 29725
+          - Gemini_Ring_J    # 490032
+    Script: |
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer))) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 2]
```diff
@@ -32236,3 +32252,1072 @@ Body:
...
+         bonus bInt,2*.@r;
+         bonus bDex,2*.@r;
+      }
+  - Combos:
+      - Combo:
+          - aegis_29725    # 29725
+          - Gemini_Shoes_J    # 470032
+    Script: |
+      .@r = getequiprefinerycnt(EQI_SHOES);
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer)))
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 3]
```diff
@@ -32236,3 +32252,1072 @@ Body:
...
+         bonus bAtkRate,3*.@r;
+         bonus bMatkRate,3*.@r;
+      }
+  - Combos:
+      - Combo:
+          - aegis_29725    # 29725
+          - Gemini_Manteau_J    # 480026
+    Script: |
+      .@r = getequiprefinerycnt(EQI_GARMENT);
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer))) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 4]
```diff
@@ -32236,3 +32252,1072 @@ Body:
...
+      .@r = getequiprefinerycnt(EQI_HEAD_TOP);
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer)))
+         bonus bPerfectHitAddRate,2*.@r;
+  - Combos:
+      - Combo:
+          - aegis_29725    # 29725
+          - Gemini_Mail_J    # 450130
+    Script: |
+      .@r = getequiprefinerycnt(EQI_ARMOR);
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer))) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 5]
```diff
@@ -32236,3 +32252,1072 @@ Body:
...
+    Script: |
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer)))
+         bonus2 bSkillAtk,"WM_REVERBERATION",150;
+  - Combos:
+      - Combo:
+          - aegis_29725    # 29725
+          - Gemini_Diadem    # 5569
+    Script: |
+      .@r = getequiprefinerycnt(EQI_HEAD_TOP);
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer)))
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 6]
```diff
@@ -32236,3 +32252,1072 @@ Body:
...
+      }
+  - Combos:
+      - Combo:
+          - aegis_29725    # 29725
+          - Gemini_Violin_J    # 570004
+      - Combo:
+          - aegis_29725    # 29725
+          - Gemini_Rope_J    # 580004          
+    Script: |
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer)))
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -161208,6 +163219,28 @@ Body:
...
+    Locations:
+      Both_Accessory: true
+    ArmorLevel: 1 
+    Script: |
+      bonus bHit,50;
+      bonus bMaxSP,100;
+      bonus bDelayrate,-5;
+      bonus bAtkRate,5;
+      bonus bMatkRate,5;
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer))) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml [hunk 8]
```diff
@@ -158627,6 +160484,63 @@ Body:
...
+    Script: |
+      .@r = getrefine();
+      bonus bMdef,5;
+      bonus bBaseAtk,20;
+      bonus bMatk,20;
+      bonus bMaxHPrate,10;
+      bonus bMaxSPrate,10;
+      bonus bDelayrate,-5;
+      bonus2 bSubEle,Ele_Neutral,20;
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer))) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml [hunk 9]
```diff
@@ -156168,6 +157855,34 @@ Body:
...
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bMdef,5;
+      bonus bBaseAtk,50;
+      bonus bMatk,50;
+      bonus bMaxHPrate,10;
+      bonus bMaxSPrate,10;
+      bonus bDelayrate,-10;
+      if ((eaclass()&EAJL_THIRD && BaseJob == Job_Bard) || ((eaclass()&EAJL_THIRD && BaseJob == Job_Dancer))) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

**Comment by Atemo** [comment]
Thanks again <3

**Pull request merged**